.pre-commit-config.yaml

```diff
@@ -1,8 +1,33 @@
 repos:
-  # Autoformat: Shell files
+  # C++
   - repo: local
     hooks:
     - id: format
       name: format
       entry: bash ./scripts/format.sh
       language: system
+      pass_filenames: false
```

Comment:
There were some strange in the pre-commit CI run and I realized by default, it runs each hook on every file it catches in the diff, so format.sh would run x times (also locally when committing btw). `pass_filename: false` fixes that so pre-commit only runs format.sh once.